ui/pages/bridge/awaiting-signatures/awaiting-signatures.tsx

```diff
@@ -0,0 +1,152 @@
+import React, { useContext, useEffect } from 'react';
+import { shallowEqual, useSelector } from 'react-redux';
+import isEqual from 'lodash/isEqual';
+
+import {
+  isHardwareWallet,
+  getHardwareWalletType,
+} from '../../../selectors/selectors';
+import PulseLoader from '../../../components/ui/pulse-loader';
+import {
+  TextVariant,
+  JustifyContent,
+  TextColor,
+  BlockSize,
+  Display,
+  FlexDirection,
+  BackgroundColor,
+} from '../../../helpers/constants/design-system';
+import { MetaMetricsContext } from '../../../contexts/metametrics';
+import { MetaMetricsEventCategory } from '../../../../shared/constants/metametrics';
+import {
+  AvatarBase,
+  AvatarBaseSize,
+  Box,
+  Text,
+} from '../../../components/component-library';
+import {
+  getBridgeQuotes,
+  getFromChain,
+  getFromToken,
+  getToToken,
+  getToChain,
+} from '../../../ducks/bridge/selectors';
+import { useI18nContext } from '../../../hooks/useI18nContext';
+
+export default function AwaitingSignatures() {
+  const t = useI18nContext();
+  const { activeQuote } = useSelector(getBridgeQuotes, shallowEqual);
+  const fromAmount = activeQuote?.sentAmount?.amount?.toNumber();
+  const fromToken = useSelector(getFromToken, isEqual);
+  const toToken = useSelector(getToToken, isEqual);
+  const fromChain = useSelector(getFromChain, isEqual);
+  const toChain = useSelector(getToChain, isEqual);
+  const hardwareWalletUsed = useSelector(isHardwareWallet);
+  const hardwareWalletType = useSelector(getHardwareWalletType);
+  const needsTwoConfirmations = Boolean(activeQuote?.approval);
+  const trackEvent = useContext(MetaMetricsContext);
+
+  useEffect(() => {
+    trackEvent({
+      event: 'Awaiting Signature(s) on a HW wallet',
```

Comment:
I don't think this event is in our metrics spec

Reply:
This event is copied from the matching "awaiting-signatures" file in swaps: https://github.com/MetaMask/metamask-extension/blob/0265469dddd94533d4f1522cc17620d7120f7c28/ui/pages/swaps/awaiting-signatures/awaiting-signatures.js#L61